### ISSUE TITLE
Multiple Transition Sets per Schema

### ISSUE THREAD

**Issue by PaulRusuTibreanu** [subvisual/fsmx#13]
Thanks for the package. Is there a possibility to have multiple transition sets per schema, for example 

```
field(:state, :string, default: "start")
field(:second_set_state, :string, default: "start")
field(:third_set_state, :string, default: "start")
...
```
If I understand correctly, state is hard-coded here.

**Comment by naps62** [comment]
Perhaps this can be added. The use case hadn't come up to me before

The original intention was to have a single state field, which technically can be achieved by setting it to a map that includes all those keys.
I do see a benefit in allowing for multiple though

**Comment by gf3** [comment]
this would be especially useful for e-commerce objects such as orders which might have an order state along with: fulfillment state, payment state, sync state, etc...

**Comment by naps62** [comment]
Now being worked on over here: https://github.com/subvisual/fsmx/pull/16

**Comment by naps62** [comment]
Merged and released in 0.5.0
